Mobile/starter_project_matador/lib/features/profile page/presentation/widget/posts_widget.dart

```diff
@@ -0,0 +1,214 @@
+import 'package:flutter/material.dart';
+
+//The first Card in My Posts section
+class Posts extends StatefulWidget {
+  const Posts({super.key});
+
+  @override
+  State<Posts> createState() => _PostsState();
+}
+
+class _PostsState extends State<Posts> {
+  @override
+  Widget build(BuildContext context) {
+    var screenWidth = MediaQuery.of(context);
+    return Container(
+      height: screenWidth.size.height * 0.25,
+      child: Card(
+        shape: RoundedRectangleBorder(
+          borderRadius: BorderRadius.circular(25.0),
+        ),
+        child: Column(
+          children: [
+            Padding(
+              padding: const EdgeInsets.fromLTRB(30, 15, 0, 0),
+              child: Row(
+                children: [
+                  const Text(
+                    'My Posts',
+                    style: TextStyle(fontSize: 18, fontWeight: FontWeight.w600),
+                  ),
+                  SizedBox(width: screenWidth.size.width * 0.55),
+                  const Icon(
+                    Icons.grid_view,
+                    color: Color.fromARGB(255, 6, 90, 159),
+                  ),
+                  SizedBox(width: 30),
+                  const Icon(
+                    Icons.toc,
+                    color: Colors.grey,
+                    size: 30,
+                  ),
+                ],
+              ),
+            ),
+            Row(
+              children: [
+                Padding(
+                  padding: const EdgeInsets.fromLTRB(30, 10, 10, 0),
+                  child: Container(
+                    width: 100,
+                    height: 120,
+                    decoration: const BoxDecoration(
+                        image: DecorationImage(
+                          image: AssetImage('../assets/laser.jpg'),
+                          fit: BoxFit.cover,
+                        ),
+                        borderRadius: BorderRadius.all(Radius.circular(20))),
+                  ),
+                ),
+                SizedBox(width: screenWidth.size.width * 0.05),
+                Column(
+                  mainAxisAlignment: MainAxisAlignment.start,
+                  children: [
+                    Padding(
+                      padding: const EdgeInsets.fromLTRB(0, 0, 0, 10),
+                      child: Text(
+                        'BIG DATA                                  ',
+                        style: TextStyle(
+                            fontSize: 17, color: Colors.blue.withOpacity(0.6)),
+                      ),
+                    ),
+                    const Padding(
+                      padding: EdgeInsets.fromLTRB(0, 0, 0, 20),
+                      child: Text(
+                        'Why Big Data Needs Thick Data?',
```

Comment:
Declare a variable with the text as a value and use that in the Text widget. it is cleaner and better for later integration as well.